pkg/sql/regionliveness/prober.go

```diff
@@ -132,22 +168,25 @@ SELECT count(*) FROM system.sql_instances WHERE crdb_region = $1::system.crdb_in
 		}
 		return txn.KV().UpdateDeadline(ctx, readTS)
 	})
+
 }
 
 // QueryLiveness implements Prober.
 func (l *livenessProber) QueryLiveness(ctx context.Context, txn *kv.Txn) (LiveRegions, error) {
 	regionStatus := make(LiveRegions)
 	executor := l.db.Executor()
-	regionProvider := l.regionProviderFactory(txn)
-	regions, err := regionProvider.GetRegions(ctx)
-	if err != nil {
-		return nil, err
+	// Database is not multi-region so report a single region.
+	if l.cachedDBRegions == nil ||
+		!l.cachedDBRegions.IsMultiRegion() {
+		regionStatus["system"] = struct{}{}
```

Comment:
Where is the "system" region used? What do you think of returning nil for LiveRegions instead of a cachedDBRegions with a magic key in it?